ffs/api/api_storageinfo.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/ipfs/go-cid"
+	"github.com/textileio/powergate/ffs"
+	"github.com/textileio/powergate/ffs/scheduler"
+)
+
+// StorageInfo returns the information about a stored Cid. If no information is available,
+// since the Cid was never stored, it returns ErrNotFound.
+func (i *API) StorageInfo(cid cid.Cid) (ffs.StorageInfo, error) {
```

Comment:
This was `Show` before which was always hard to remember what it does. This will be easier to remember.